packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.spec.ts

```diff
@@ -186,6 +191,31 @@ describe('resourceCollection', () => {
       const traceInfo = (rawRumEvents[0].rawRumEvent as RawRumResourceEvent)._dd!
       expect(traceInfo).not.toBeDefined()
     })
+
+    it('should pull tracingSampleRate from config if present', () => {
+      setupBuilder = setup().beforeBuild(({ lifeCycle }) => {
+        startResourceCollection(
+          lifeCycle,
+          validateAndBuildRumConfiguration({
+            clientToken: 'xxx',
+            applicationId: 'xxx',
+            tracingSampleRate: 60,
+          }) as RumConfiguration
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏same here
```suggestion
          })!
```